inst/@sym/unique.m

```diff
@@ -67,3 +72,8 @@
 %! A = [1 2 3 3 4 5 5 6 7 7 x x y y];
 %! B = [1  2  3  4  5  6  7  x  y];
 %! assert (isequal (unique(A), B))
+
+%!test
+%! syms x
+%! A = [x]
```

Comment:
Maybe this is just `A = x`?

Reply:
Yep and thus it is actually not needed. We can just use `x`...